charts/thoras/tests/collector_deployment_test.yaml

```diff
@@ -40,5 +40,14 @@ tests:
       - equal:
           path: spec.template.spec.containers[0].volumeMounts
           value:
-            - mountPath: /usr/share/elasticsearch/data
+            - mountPath: /var/lib/share/elasticsearch
               name: elastic-search-data
+  - it: Should create timescale container when enabled
```

Comment:
[pebble] Pebble-level feedback so it's your choice, but for these I think it's useful to have another test that asserts it's not there when the value is omitted or false. I might be more concerned about accidentally running PG when we didn't mean to versus not running PG when we meant to

Reply:
ignore me! I see we're basically doing this above